docs/source/usage/workflow.rst

```diff
@@ -1,5 +1,38 @@
 .. _workflow:
 
+Access modes
+============
+
+The openPMD-api distinguishes between a number of different access modes:
+
+* **Create mode**: Used for creating a new Series from scratch.
+  Any file possibly existing in the specified location will be overwritten.
```

Comment:
For file-based I/O, I think there might be a potentially confusing corner case:
if one writes the same series again, but with different interval, then some files might be overwritten and others untouched.

I am thinking about this from time to time and see pro and cons for, e.g., actively cleaning/moving out all potential files (even for iterations that are not created again) when _opening_ in truncating create mode.

More of a thing we could discuss next time.

Reply:
Fair point. I guess the most sensible solution would be within the bounds of a potential truncate option.